msal/oauth2cli/authcode.py

```diff
@@ -36,25 +36,11 @@ def obtain_auth_code(listen_port, auth_uri=None):
     :param auth_uri: If provided, this function will try to open a local browser.
     :return: Hang indefinitely, until it receives and then return the auth code.
     """
-    exit_hint = "Visit http://localhost:{p}?code=exit to abort".format(p=listen_port)
-    logger.warning(exit_hint)
     if auth_uri:
-        page = "http://localhost:{p}?{q}".format(p=listen_port, q=urlencode({
```

Comment:
The latest commit(s) move this part to a test case. Is it possible to still have this functionality here? We may be able to have the presence or absence of a new parameter `obtain_auth_code(..., text="Open this link to sign in. You may use incognito window")` to toggle this test functionality.